crates/experimental-api/Cargo.toml

```diff
@@ -18,10 +18,15 @@ tracing.workspace = true
 thiserror.workspace = true
 
 starknet-crypto = "0.6.1"
-blockifier = { git = "https://github.com/starkware-libs/blockifier", tag = "v0.5.0-rc.2" }
-starknet_api = "0.8.0"
+blockifier = { git = "https://github.com/starkware-libs/blockifier", tag = "v0.5.0" }
```

Comment:
Why not `blockifier = "0.5.0"`?

Reply:
For some reason I was sure it wasn't available on crates.io. https://github.com/eigerco/beerus/pull/638